### PR TITLE
Create kubeadm-alpha.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
> 
> For pull requests on Chinese localization, set Base Branch to release-1.15
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+title: kubeadm alpha
+weight: 90
+---
+-->
+{{< caution >}}
+<!--
+`kubeadm alpha` provides a preview of a set of features made available for gathering feedback
+ from the community. Please try it out and give us feedback!
+-->
+ `kubeadm alpha`提供了一组来自社区的可用于收集反馈的功能都预览。请试试看，并给我们反馈！
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
`kubeadm alpha` 提供了一组可用于收集社区反馈的功能的预览。请尝试一下，并给我们反馈!
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+{{< /caution >}}
+
+<!--
+## kubeadm alpha certs renew {#cmd-certs-renew}
+-->
+## kubeadm alpha 证书更新{#cmd-certs-renew}
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
## kubeadm alpha 证书更新 {#cmd-certs-renew}
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+<!--
+You can renew all Kubernetes certificates using the `all` subcommand or renew them selectively.
+-->
+您可以使用`all`子命令更新所有 Kubernetes 证书，或者有选择的更新它们。
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
您可以使用 `all` 子命令更新所有 Kubernetes 证书，或者有选择的更新它们。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+{{< tab name="etcd-peer" include="generated/kubeadm_alpha_certs_renew_etcd-peer.md" />}}
+{{< tab name="etcd-server" include="generated/kubeadm_alpha_certs_renew_etcd-server.md" />}}
+{{< tab name="front-proxy-client" include="generated/kubeadm_alpha_certs_renew_front-proxy-client.md" />}}
+{{< /tabs >}}
+
+
+<!--
+## kubeadm alpha kubeconfig user {#cmd-phase-kubeconfig}
+-->
+## kubeadm alpha kubeconfig 用户{#cmd-phase-kubeconfig}
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
## kubeadm alpha kubeconfig 用户 {#cmd-phase-kubeconfig}
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+<!--
+The `user` subcommand can be used for the creation of kubeconfig files for additional users.
+-->
+`user`子命令可用于为其他用户创建 kubeconfig 文件。
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
`user` 子命令可用于为其他用户创建 kubeconfig 文件。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+<!--
+## kubeadm alpha selfhosting pivot {#cmd-selfhosting}
+-->
+## kubeadm alpha 自托管 pivot {#cmd-selfhosting}
+
+<!--
+The subcommand `pivot` can be used to conver a static Pod-hosted control plane into a self-hosted one.
+-->
+`pivot`子命令可用于将静态的 Pod 托管控制平面转换为自托管控制平面。
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
`pivot` 子命令可用于将静态的 Pod 托管控制平面转换为自托管控制平面。
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+{{< tabs name="selfhosting" >}}
+{{< tab name="selfhosting" include="generated/kubeadm_alpha_selfhosting.md" />}}
+{{< tab name="pivot" include="generated/kubeadm_alpha_selfhosting_pivot.md" />}}
+{{< /tabs >}}
+
+
+<!--
+## What's next
+-->
+## 接下来
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
## 下一步
```

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+<!--
+* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init/) to bootstrap a Kubernetes control-plane node
+* [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join/) to connect a node to the cluster
+* [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset/) to revert any changes made to this host by `kubeadm init` or `kubeadm join`
+-->
+* [kubeadm init](/docs/reference/setup-tools/kubeadm/kubeadm-init/) 来用 bootstrap 启动一个 Kubernetes 控制平面节点
+* [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join/) 将节点连接到集群
+* [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset/) 使用 `kubeadm init` 或 `kubeadm join`恢复对主机所做的任何更改
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-alpha.md]:
```suggestion
* [kubeadm reset](/docs/reference/setup-tools/kubeadm/kubeadm-reset/) 使用 `kubeadm init` 或 `kubeadm join` 恢复对主机所做的任何更改
```

**Comment by xichengliudui** [comment]
@zhangx501  Thank you for your contribution, please troubleshoot the build problems and fix them.

**Comment by markthink** [comment]
@zhangx501  建议本地构建一下，解决构建问题

**Comment by markthink** [comment]
@zhangx501  冲突需要抽空解决一下，谢谢

**Comment by tengqm** [comment]
Closing due to long period of no action. Please feel free to reopen when time permits.

**Pull request closed without merging**